control/installation.xml

```diff
@@ -109,7 +109,7 @@
 		  <!-- we do need most likely more repos with "base ofed"-->
                   <default_patterns>minimal_base hpc_compute_node hpc_libraries ofed file_server directory_server hpc_development_node yast2_basis devel_basis</default_patterns>
                   <clone_install_recommended_default config:type="boolean">true</clone_install_recommended_default>
-                  <minimalistic_libzypp_config config:type="boolean">true</minimalistic_libzypp_config>
+                  <minimalistic_libzypp_config config:type="boolean">false</minimalistic_libzypp_config>
```

Comment:
if it is backed by some decision I suggest to you to add comment above it with reference to that bug, so in future you know WHY you have this settings and can revisit decision if needed.